### PR TITLE
imported header and footer templates from o-header next branch

### PR DESCRIPTION
**Pull request by wheresrhys**
@aelias @keirog @ironsidevsquincy

### CODE REVIEW

**Comment by wheresrhys** [comment]
See also https://github.com/Financial-Times/o-header/pull/141

Waiting on creation of o-icons component too

File: main.scss [hunk 1]
```diff
@@ -20,9 +20,21 @@
     }
 }
 
+@mixin unsetNLinksBody($hoverColor: blue) {
```
Comment (keirog) [inline main.scss]: How do you feel about hyphenating `hoverColor`? Would be more consistent with other Sass variables

File: main.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 module.exports.init = function init(flags, opts = {}) {
-	require('o-header').init(flags);
+	require('./origami/header/main').init(flags);
```
Comment (keirog) [inline main.js]: Can you explain the `./origami/` bit?

File: origami/header/main.scss [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+}
+.o-header__meganav-link,
+.o-header__typeahead-link {
+	@include unsetNLinksBody(inherit);
+}
+
+@import "o-header/main";
+
+.o-header__typeahead {
+	list-style: none;
```
Comment (keirog) [inline origami/header/main.scss]: Are these in a particular order?

File: origami/header/main.scss [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+	position: absolute;
+	left: 0;
+	right: 0;
+	top: 45px;
+	border: 0;
+	background-color: #fcfcfc;
+	color: oColorsGetPaletteColor('grey-tint5');
+	z-index: 11;
```
Comment (keirog) [inline origami/header/main.scss]: Could we use `zIndexFor` please

**Comment by keirog** [comment]
Apart from comments, LGTM

**Comment by wheresrhys** [comment]
now #70

**Pull request closed without merging**